Velocity4LumiContainers/images/fakelumig/templates/opensuse.vtmp

```diff
@@ -0,0 +1,23 @@
+@from
+    {{ __base__ }}
+
+@copy
+    rocm_agent_enumerator rocm_agent_enumerator
+    rocminfo rocminfo
+    amdgpu-arch.cpp amdgpu-arch.cpp
+
+@run
+    >>> Replace rocm information
```

Comment:
I think we should consider adding a bit more description about what this `fakelumig` image does, either are more elaborate comments or as [labels](https://olcf.github.io/velocity/reference/vtmp.html#label).